compiler/damlc/tests/daml-test-files/ExceptionSemantics.daml

```diff
@@ -2,10 +2,13 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 -- @SINCE-LF-FEATURE DAML_EXCEPTIONS
--- @ERROR range=145:1-145:11; Attempt to exercise a consumed contract
+-- @ERROR range=130:1-130:23; Unhandled exception: ExceptionSemantics:E
+-- @ERROR range=145:1-145:25; CRASH: ArithmeticError while evaluating DIV_INT64(1,0).
```

Comment:
Now you just added an assertion that we hit the `crash` instead of fixing the problem. Users should never see a `crash` for a type-correct daml-lf program. `crash` is for internal errors.

Reply:
852de80ac6ad778835986b78094512feb8a39d53